tensorflow/docs_src/install/install_sources.md

```diff
@@ -76,9 +76,9 @@ issue the following command:
 <pre>$ <b>git checkout r1.0</b></pre>
 
 Next, you must prepare your environment for
-[Linux](#PrepareLinux)
+[Linux](#prepare_environment_for_linux)
 or
-[macOS](#PrepareMac)
+[macOS](#prepare_environment_for_macos)
 
 
 <a name="#PrepareLinux"></a>
```

Comment:
Not sure I understand. Isn't the link name set here?

Reply:
Oh I didn't notice the link name here. But actually it not works as you can see above links event it has set this link name. 

I mean https://www.tensorflow.org/install/install_sources#PrepareLinux cannot direct to the correct anchor section but https://www.tensorflow.org/install/install_sources#prepare_environment_for_linux can.

I can remove this link name "PrepareLinux" line or keep this but fix in "PrepareLinux" way. Is there any preference for you? If either would be ok, I'll remove this "PrepareLinux" line.

Reply:
@jhseu Ah I found why the pre-defined anchor not works, it has a wrong # in the anchor name. Have pushed a fix to fix this incorrect anchor name.